site/docs/master/supported-providers.md

```diff
@@ -4,20 +4,21 @@ Velero supports a variety of storage providers for different backup and snapshot
 
 ## Velero supported providers
 
-| Provider                   | Object Store        | Volume Snapshotter           | Plugin                    |
-|----------------------------|---------------------|------------------------------|---------------------------|
-| [AWS S3][7]                | AWS S3              | AWS EBS                      | [Velero plugin AWS][8]    |
-| [Azure Blob Storage][9]    | Azure Blob Storage  | Azure Managed Disks          | [Velero plugin Azure][10] |
-| [Google Cloud Storage][11] | Google Cloud Storage| Google Compute Engine Disks  | [Velero plugin GCP][12]   |
+| Provider                          | Object Store        | Volume Snapshotter           | Plugin Documentation      |
+|-----------------------------------|---------------------|------------------------------|---------------------------|
+| [Amazon Web Services (AWS)][7]    | AWS S3              | AWS EBS                      | [Velero plugin AWS][8]    |
```

Comment:
Hm. I didn't originally add the `for` but looking at it now I think we should. Ex; `Velero plugin for AWS`.

Reply:
sure, I can add that